plugins/lookup/onepassword.py

```diff
@@ -105,12 +105,12 @@
 from ansible.errors import AnsibleLookupError
 from ansible.module_utils.common.text.converters import to_bytes, to_text
 
+from ansible_collections.community.general.plugins.module_utils.onepassword import OnePasswordConfig
 
-class OnePass(object):
 
+class OnePass:
```

Comment:
I think this should stay for Python 2.7 compatibility.
```suggestion
class OnePass(object):
```
(Similar ones below.)

Reply:
I wasn't sure if we still need to maintain Python 2.7 compatibility for lookups. But it doesn't hurt to keep ensure new style modules get used in Python 2.7, so I'll put it back in.

Reply:
ansible-core 2.11 still supports Python 2.7 on the controller, and since we'll still be supporting that one for a bit of time, let's keep it until we drop support for it. (Files in module_utils will need it a lot longer since they can be used by modules, and on the module side ansible-core is still supporting 2.7.)